test/torchaudio_unittest/datasets/tedlium_test.py

```diff
@@ -145,5 +151,6 @@ class TestTedliumSoundfileNew(Tedlium, TorchaudioTestCase):
     class TestTedliumSox(Tedlium, TorchaudioTestCase):
         backend = "sox"
 
+
```

Comment:
Looks like this is causing style test to fail, though I think this is okay. Can you fix this?